src/Storages/System/StorageSystemDistributionQueue.cpp

```diff
@@ -0,0 +1,116 @@
+#include <Columns/ColumnString.h>
```

Comment:
Looks like we need to add this file to `Storages/ya.make`

Reply:
Hm, don't see that other files from `Columns/` added, how it should be added as `../Columns/ColumnString.h`?

Reply:
I'm about src/Storages/System/StorageSystemDistributionQueue.cpp.

Reply:
It is already there, and that's why I don't have any ideas on how to fix that internal check

Reply:
Ah, ok. It's just false failure.